content/concepts/feathersjs.textile

```diff
@@ -268,4 +268,8 @@ The problem with adding load balancers and multiple servers is the associated co
 
 "Ably’s engineering blog":https://www.ably.io/blog/tag/ably-engineering/ describes problems and solutions associated with scaling realtime apps. When you find your (or your team’s) time taken up solving and/ or provisioning for these issues, that’s when - we believe - it’s worth looking for a cloud or better still serverless solution, leaving you free to concentrate on the core functionality of your app. That said, for small-scale applications or developers starting out with realtime - until realtime functionality scales and gets complicated, there is indeed a point at which FeathersJS could fit the bill of the holy grail of frameworks. Developers can judge for themselves when holy grail status starts to fade, and think about how to move on.
 
-*"Ably":https://www.ably.io/ is a global cloud network for streaming data and managing the full lifecycle of realtime APIs. To talk about scaling your app or service, get in touch with our dev-rel team.*
\ No newline at end of file
+"Ably":https://ably.com is a realtime messaging platform. We deliver billions of realtime messages every day to more than 50 million end-users across web, mobile, and IoT platforms.
```

Comment:
@kennethkalmer are we wanting these to link to ably.com already, or keep to ably.io until we properly transition?

Reply:
@tomczoink @Nosfheratu lets keep it to `.io` for now.

Reply:
@kennethkalmer should @Nosfheratu go through and change these to .io now so we can land this PR?